.github/workflows/update-infrastructure-docker.yaml

```diff
@@ -0,0 +1,31 @@
+name: Update infrastructure-docker submodule
```

Comment:
did you already make a PR to infrastructure-docker to add the submodule?
otherwise this action will fail when merged

Reply:
Added a PR: https://github.com/MiSArch/infrastructure-docker/pull/162